Zend/zend_fibers.c

```diff
@@ -457,17 +474,8 @@ ZEND_METHOD(Fiber, __construct)
 	Z_TRY_ADDREF(fiber->fci.function_name);
 }
 
-ZEND_METHOD(Fiber, start)
+ZEND_API void zend_fiber_start(zend_fiber *fiber, zval *params, uint32_t param_count, zend_array *named_params, INTERNAL_FUNCTION_PARAMETERS)
```

Comment:
I don't think exported APIs should be using INTERNAL_FUNCTION_PARAMETERS. What do you need it for here?

Reply:
Switched to only passing return_value and switched to using EG(current_execute_data).